src/bionics.cpp

```diff
@@ -1693,13 +1693,10 @@ void Character::process_bionic( bionic &bio )
             mod_power_level( -trigger_cost );
         }
     } else if( bio.id == bio_gills ) {
-        const units::energy trigger_cost = bio.info().power_trigger / 8;
-        if( has_effect( effect_asthma ) && get_power_level() >= trigger_cost ) {
+        if( has_effect( effect_asthma ) ) {
             add_msg_if_player( m_good,
-                               _( "Your %s activates and you feel your throat open up and air filling your lungs!" ),
-                               bio.info().name );
+                               _( "You feel your throat open up and air filling your lungs!" ) );
             remove_effect( effect_asthma );
```

Comment:
```suggestion
            remove_effect( effect_asthma );
            mod_power_level( -trigger_cost );
```

Reply:
How would I know? I haven't changed it and I don't know where it comes from. The tools haven't flagged it as needing conflict resolution, so I haven't had any reason to change it back. I have not opinion about changes to that code in either direction, so I don't have any problem accepting suggested changes.